.github/workflows/tiledb-cloud-py.yaml

```diff
@@ -0,0 +1,51 @@
+name: TileDB-Cloud-Py autobuild
+
+on:
+  push:
+    tags: [ "v*" ]
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Do you want to run on all pull requests instead of only master? That way if you do a PR into a different branch it runs?

Reply:
Good idea. Changed.